src/user/wikis/wikis.scss

```diff
@@ -0,0 +1,10 @@
+.wikis {
+  h2 {
+    font-size: 20px;
+    font-weight: bold;
+    font-family: "Inter";
+  }
+  .btn-primary {
+    background-color: #1a73e8 !important;
```

Comment:
Not good practice

Reply:
@Rupeshiya Plz try not to open multiple comments for the same thing, One comment with the note to do it across the PR would have sufficed.

Reply:
Ok sorry, will keep that in mind,

Reply:
Will not do from next time.
Thanks

Reply:
Done that bcoz it didn't increase my work.

Reply:
@Rupeshiya If it's fine with you, then its actually great for me as I get to know exactly where I have to make the changes. I value my reviewer's time so I don't want them to do the extra work for me, so that was just a humble suggestion. I hope you understand.

Reply:
Thanks

Reply:
Done